client/src/helpers/router/navigation.js

```diff
@@ -274,6 +287,12 @@ class Navigation {
         route: "backendRecordsFeatures",
         entity: "feature",
         ability: "update"
+      },
+      {
+        label: "Export Targets",
+        route: "backendRecordsExportTargets",
```

Comment:
I noticed under the Records label on line 57 that there was an array of objects that are also defined here. Am I doing this wrong by adding the Export Targets route information in two places on the same file?

Reply:
This file is confusing—I'm not sure why it was built the way it was. I think what you did is fine, but we need to refactor this navigation at some point.